release-notes/9.0/supported-os.md

```diff
@@ -0,0 +1,128 @@
+[OpenSUSE-pm]: https://learn.microsoft.com/dotnet/core/install/linux-package-manager-opensuse15
+[Oracle-Linux]: https://www.oracle.com/linux/
+[Oracle-Lifecycle]: https://www.oracle.com/a/ocom/docs/elsp-lifetime-069338.pdf
+[RHEL]: https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux
+[RHEL-lifecycle]: https://access.redhat.com/support/policy/updates/errata/
+[RHEL-msft-pm]: https://learn.microsoft.com/dotnet/core/install/linux-package-manager-rhel8
+[RHEL-pm]: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/developing_.net_applications_in_rhel_8/using-net-core-on-rhel_gsg#installing-net-core_gsg
+[SLES]: https://www.suse.com/products/server/
+[SLES-lifecycle]: https://www.suse.com/lifecycle/
+[SLES-pm]: https://learn.microsoft.com/dotnet/core/install/linux-package-manager-sles15
+[Ubuntu]: https://ubuntu.com/
+[Ubuntu-lifecycle]: https://wiki.ubuntu.com/Releases
+[Ubuntu-pm]: https://learn.microsoft.com/dotnet/core/install/linux-package-manager-ubuntu-2004
+[glibc]: https://www.gnu.org/software/libc/
+[musl]: https://musl.libc.org/
+
+## macOS
+
+OS                            | Version                   | Architectures     |
+------------------------------|---------------------------|-------------------|
+[macOS][macOS]                | 12.0+                     | x64, Arm64        |
+
+.NET 8 is supported in the Rosetta 2 x64 emulator.
+
+[macOS]: https://support.apple.com/macos
+
+## Android
+
+OS                            | Version                 | Architectures     |
+------------------------------|-------------------------|-------------------|
+[Android][Android]            | API 21+                 | x64, Arm32, Arm64 |
+
+[Android]: https://support.google.com/android
+
+## iOS / tvOS / MacCatalyst
+
+OS                            | Version                 | Architectures     |
+------------------------------|-------------------------|-------------------|
+[iOS][iOS]                    | 12.0+                   | Arm64             |
+[iOS Simulator][iOS]          | 12.0+                   | x64, Arm64        |
+[tvOS][tvOS]                  | 12.0+                   | Arm64             |
+[tvOS Simulator][tvOS]        | 12.0+                   | x64, Arm64        |
+[MacCatalyst][macOS]          | 12.0+                   | x64, Arm64        |
+
+[iOS]: https://support.apple.com/ios
+[tvOS]: https://support.apple.com/apple-tv
+
+## QEMU
+
+.NET 8 is not supported being run (emulated) via [QEMU](https://www.qemu.org/). QEMU is used, for example, to emulate Arm64 containers on x64, and vice versa.
+
+## Support changes from .NET 6.0
+
+The following operating systems are no longer supported, starting with .NET 7.0.
+
+OS                                    | Version                 | Architectures     |
+--------------------------------------|-------------------------|-------------------|
+[Windows Client][Windows-client]      | 7 SP1, 8.1      | x64, x86          |
```

Comment:
I don't think we need this content. Also, the references to .NET 6 and 7 are confusing.

```suggestion
```